### PR TITLE
Adding 2nd Org to Fabric Testnet

### PR DESCRIPTION
**Pull request by sanvenDev**

### CODE REVIEW

File: tests/network-setups/fabric/dev/docker/docker-compose-ca.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ networks:
 
 services:
   ca_org1:
+    profiles: ["1-node", "2-nodes"]
```
Comment (VRamakrishna) [inline tests/network-setups/fabric/dev/docker/docker-compose-ca.yaml:13]: We are always going to start this service, right? If we forget to supply a profile ID, it won't start. So I think it's safer to remove this line.

File: tests/network-setups/fabric/dev/docker/docker-compose-couch.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ networks:
 
 services:
   couchdb0:
+    profiles: ["1-node", "2-nodes"]
```
Comment (VRamakrishna) [inline tests/network-setups/fabric/dev/docker/docker-compose-couch.yaml:13]: Same comment as above.

File: tests/network-setups/fabric/dev/docker/docker-compose-couch.yaml [hunk 3]
```diff
@@ -20,18 +21,49 @@ services:
     # Comment/Uncomment the port mapping if you want to hide/expose the CouchDB service,
     # for example map it to utilize Fauxton User Interface in dev environments.
     ports:
-      - "${COUCHDB_PORT}:${COUCHDB_PORT}"
+      - "${COUCHDB0_PORT}:${COUCHDB0_PORT}"
     networks:
       - net
 
   peer0.org1.${COMPOSE_PROJECT_NAME}.com:
+    profiles: ["1-node", "2-nodes"]
```
Comment (VRamakrishna) [inline tests/network-setups/fabric/dev/docker/docker-compose-couch.yaml:29]: Same comment as above.

File: tests/network-setups/fabric/dev/docker/docker-compose-test-net.yaml [hunk 4]
```diff
@@ -48,6 +49,7 @@ services:
       - net
 
   peer0.org1.${COMPOSE_PROJECT_NAME}.com:
+    profiles: ["1-node", "2-nodes"]
```
Comment (VRamakrishna) [inline tests/network-setups/fabric/dev/docker/docker-compose-test-net.yaml:52]: Same comment as above.

File: tests/network-setups/fabric/dev/network.sh [hunk 5]
```diff
@@ -279,8 +280,8 @@ function createOrgs() {
   fi
 
   echo
-  #echo "Generate CCP files for Org1 and Org2"
-  echo "Generate CCP files for Org1 in $NW_CFG_PATH"
+  echo "Generate CCP files for Org1 and Org2 in $NW_CFG_PATH"
+  # echo "Generate CCP files for Org1 in $NW_CFG_PATH"
```
Comment (VRamakrishna) [inline tests/network-setups/fabric/dev/network.sh:284]: Remove this line.

File: tests/network-setups/fabric/dev/scripts/createChannel.sh [hunk 6]
```diff
@@ -7,128 +7,127 @@ MAX_RETRY="$3"
...
+    set +x
+    if [ $res -ne 0 ]; then
+        echo "Failed to generate channel configuration transaction..."
+        exit 1
+    fi
+    echo
 
 }
 
 createAncorPeerTx() {
```
Comment (VRamakrishna) [inline tests/network-setups/fabric/dev/scripts/createChannel.sh:42]: Typo: `Ancor` --> `Anchor`.

File: tests/network-setups/fabric/dev/scripts/createChannel.sh [hunk 7]
```diff
@@ -140,7 +139,7 @@ createChannelTx
 
 ## Create anchorpeertx
 echo "### Generating channel configuration transaction '${CHANNEL_NAME}.tx' ###"
-createAncorPeerTx
+createAncorPeerTx "Org1MSP Org2MSP"
```
Comment (VRamakrishna) [inline tests/network-setups/fabric/dev/scripts/createChannel.sh:142]: Typo: `Ancor` --> `Anchor`.

File: tests/network-setups/fabric/dev/docker/docker-compose-ca.yaml (hunk 1 above)
Comment (sanvenDev) [reply tests/network-setups/fabric/dev/docker/docker-compose-ca.yaml:13]: Based on script default value is `1-node` (i.e. if no profile is specified, this value will be taken), so if we start via script it won't be the case that profile ID is empty.

Comment (sanvenDev) [reply tests/network-setups/fabric/dev/docker/docker-compose-ca.yaml:13]: I can remove it, if you feel this is redundant, just thought to it'd be verbose.

Comment (VRamakrishna) [reply tests/network-setups/fabric/dev/docker/docker-compose-ca.yaml:13]: Yeah, I think it'd be better to remove it, so we don't depend on a well-written script. It's more maintainable that way.

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**